dex/market.go

```diff
@@ -5,17 +5,19 @@ package dex
 
 import (
 	"fmt"
+	"math"
 	"strings"
 )
 
 // MarketInfo specifies a market that the Archiver must support.
 type MarketInfo struct {
```

Comment:
Shouldn't we move this type to **server/db**, or is there some kind of circular dependency thing?

Reply:
Hmm, I suppose it could.  Although having it in `dex` keeps certain package's deps really tight.  e.g. server/cmd/dcrdex only requires `decred.org/dcrdex/dex` and `decred.org/dcrdex/server/dex`.  We can consider moving this.  I'm a little hesitant at the moment though.